test-tools/wamr-ide/VSCode-Extension/README.md

```diff
@@ -3,10 +3,13 @@
 ### An integrated development environment for WASM.
 
 # How to debug this extension
-
-> Note that please build `lldb` firstly follow this
-> [instruction](./resource/debug/README.md) if you want to enable
-> `source debugging` feature of this extension
+> Note that when you download and 
+> decompress to get .vsix file from [our release](https://github.com/bytecodealliance/wasm-micro-runtime/releases). 
+> It's by default that the `source debugging` feature is not enabled. 
+> If you want to enable `source debugging` feature of this extension,
+> you could either download `lldb` from [our release](https://github.com/bytecodealliance/wasm-micro-runtime/releases) and put them in correct path
+mentioned in this [instruction](./resource/debug/README.md) (This is recommended way), 
```

Comment:
can this be automatic?

Reply:
is it common for a vscode extension to download something like this?
(just a question. i'm not familiar with vscode.)

Reply:
Yes, it's common for vscode extension to download platform dependent binaries, (e.g. [Microsoft's cpp-tools](https://github.com/microsoft/vscode-cpptools) also download the debugger for the target platform after installation)

Reply:
ok. thank you for the info.